packages/syft/src/syft/service/action/action_object.py

```diff
@@ -700,11 +706,15 @@ class ActionObjectV2(SyftObject):
 
 
 @serializable(without=["syft_pre_hooks__", "syft_post_hooks__"])
-class ActionObject(SyftObject):
+class ActionObject(SyncableSyftObject):
     """Action object for remote execution."""
 
     __canonical_name__ = "ActionObject"
     __version__ = SYFT_OBJECT_VERSION_3
+    __private_sync_attrs__: ClassVar[dict[str, Any]] = {
```

Comment:
I think this name is slightly confusing, because its actually not the private version of this data. Maybe __private_sync_attr_mocks__?